apps/tenant-management-webapp-e2e/src/integration/directory-service.feature

```diff
@@ -142,4 +142,48 @@ Feature: Directory-service
     Then the user "views" Add entry modal
     And no critical or serious accessibility issues for "directory entry modal" on "directory entries page"
     When the user clicks Cancel button in Entry modal
-    Then the user "should not view" Add entry modal
\ No newline at end of file
+    Then the user "should not view" Add entry modal
+
+  @TEST_CS-4002 @REQ_CS-3616 @TEST_CS-4018 @REQ_CS-3618 @TEST_CS-4017 @REQ_CS-3617 @TEST_CS-3988 @REQ_CS-3615 @regression
+  Scenario:  As a tenant admin, I can add/edit/delete/view configured resources types
+    Given a tenant admin user is on resource types page
```

Comment:
Given a tenant admin user is on resource types page
Should be Directory service page